app/views/cases/cover_pages/_data_requests.html.slim

```diff
@@ -1,7 +1,9 @@
 .grid-row
   .column-full.data-requests
 
-    - if case_details.data_requests.empty?
+    - if case_details.current_state == "rejected"
```

Comment:
Would it be better to remove the cover page functionality entirely from a rejected case?